cli/src/find_path.rs

```diff
@@ -0,0 +1,47 @@
+use super::pager::Pager;
+use asset_identifier;
+use clap::ArgMatches;
+use error::Result;
+use fmt::{Formatter, Simple};
+use std::str::FromStr;
+use stellar_client::{sync, endpoint::payment, resources::Amount, sync::Client};
+
+pub fn find_path(client: &Client, matches: &ArgMatches) -> Result<()> {
+    let pager = Pager::from_arg(&matches);
+
+    let endpoint = {
+        let destination_account = matches
+            .value_of("to")
+            .expect("Destination account is a required field");
+        let source_account = matches
+            .value_of("from")
+            .expect("Source account is a required field");
+        let destination_amount = matches
+            .value_of("amount")
+            .expect("Destination amount is a required field");
+        let destination_asset_str = matches
+            .value_of("asset")
+            .expect("Destination asset is a required field");
+        let destination_asset = asset_identifier::from_str(destination_asset_str)?;
+        let endpoint = payment::FindPath::new(
+            source_account,
+            destination_account,
+            destination_asset,
+            Amount::from_str(destination_amount).unwrap(),
```

Comment:
I generally like to use `expect` here instead so that you can provide a message. Of course, we should really be returning an Result from this path and allowing it to be handled by the outer logic.

Reply:
Would this be more appropriate -- to parse and throw an error as part of the argument matching process? Although I admit the double expect does not seem right.

```rust
let destination_amount: Amount = matches
            .value_of("amount")
            .expect("Destination amount is a required field")
            .parse()
            .expect("Amount must be properly formatted");
```

This would then allow:
```rust
let endpoint = payment::FindPath::new(
            source_account,
            destination_account,
            destination_asset,
            destination_amount,
        );
```

Reply:
Yup, that would be better I think. I've been recommending just returning an `Error` so that exit is centralized, but it's a CLI so it's not the end of the world.

Eventually, centralized error reporting, would provide a consistently formatted error text though.